osu.Game.Rulesets.Catch/Objects/JuiceStream.cs

```diff
@@ -76,6 +76,8 @@ protected override void CreateNestedHitObjects(CancellationToken cancellationTok
         {
             base.CreateNestedHitObjects(cancellationToken);
 
+            this.PopulateNodeSamples();
```

Comment:
So is every single implementation of `CreateNestedHitObjects` going to have to call this?

Reply:
Every `IHasRepeats` would need to I suppose. Not entirely sure how to fix that without introducing eldritch abstract entities.

Reply:
@smoogipoo do you have strong opinions on this?

Reply:
not at the moment, no